### DISCUSSION TITLE
Hiding profile activity should still show pinned repositories and organization memberships

### DISCUSSION THREAD

**Discussion by amarshall** [community/community#28771] [Other Feature Feedback, Questions, & Ideas]
This is regarding the new “Make profile private and hide activity” option.

Pinned repositories and visible organizations aren’t “activity”, and can already be hidden (separately from this option). Changing to not force-hide them when enabling the above option would be in-line with how the option treats other already-removable, non-activity that serves little value to the account owner themselves (e.g. profile picture, description, location, website).

**Comment by kubk** [comment]
Just wanted to create this request by myself but decided to search for it first, just in case. I was very surprised when my repositories disappeared, they also do not seem like an activity for me

UPD: After a more thorough search I've found that actually the issue is being discussed here: https://github.com/community/community/discussions/15365
